07LessonSeven/homework/css/style.css

```diff
@@ -74,6 +74,8 @@ p {
   background-color: #55a747
 }
 
+/*Aspects for Both buttons*/
+
 input {
   background-color: #313131;
   border: 1px solid white;
```

Comment:
Great use of flexbox, thank you! `margin: 0px 0px 0px 0px;` this could have been shorten to `margin: 0`. Remember those shorthand CSS properties when the same value needs to be set for all 4 locations (top, right, bottom and left). Also, you don't add size units when using `0` as value, so don't do this `margin: 0px` do this instead `margin: 0` :smiley: